docs/reference/esql/index.asciidoc

```diff
@@ -92,6 +92,11 @@ POST /_query?format=txt
 ----
 // TEST[setup:library]
 
+The above query's `LIMIT` command limits results to 5 rows.
+
+If not specified, `LIMIT` defaults to `500`. A single query can't return more
```

Comment:
Only a comment, can be safely ignored. `can't` => `cannot` OR `A single query will not return more....` (there is a slight difference between "not being able to" and "not being allowed/not wanting to").